sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -50,16 +48,44 @@ The API key can be found in the [Azure Portal][azure_portal] or by running the f
 ```bash
 az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
 ```
+#### Create a ContentSafetyClient with Azure Active Directory (AAD) token credential
+To use an [Azure Active Directory (AAD) token credential](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity)
```

Comment:
The section about creating a ContentSafetyClient with Azure Active Directory (AAD) token credential is explaining the process using JavaScript-related information (e.g., npm install), which is not relevant to a Java library. This section needs to be revised to correctly explain how to authenticate using AAD in Java.

The links are all `azure-sdk-for-js`

Reply:
added it back